var/spack/repos/builtin/packages/py-llvmlite/package.py

```diff
@@ -28,13 +29,14 @@ class PyLlvmlite(PythonPackage):
     version("0.23.0", sha256="bc8b1b46274d05b578fe9e980a6d98fa71c8727f6f9ed31d4d8468dce7aa5762")
 
     depends_on("py-setuptools", type="build")
+    depends_on("python@3.8:", type=("build", "run"), when="@0.40.0:")
     depends_on("python@3.7:3.10", type=("build", "run"), when="@0.38.0:")
     depends_on("python@3.7:3.9", type=("build", "run"), when="@0.37")
     depends_on("python@3.6:", type=("build", "run"), when="@0.33:")
     depends_on("python@2.6:2.8,3.4:", type=("build", "run"))
 
     # llvmlite compatibility information taken from https://github.com/numba/llvmlite#compatibility
-    depends_on("llvm@11.0:11~flang", when="@0.37.0:")
+    depends_on("llvm@11.0:14~flang", when="@0.37.0:")
```

Comment:
The table at https://github.com/numba/llvmlite#compatibility just seems incorrect. LLVM 12-14 support was only [added](https://github.com/numba/llvmlite/commit/7b977a6ff830356dba253ba9b18a070f79df75b7) in 0.40.
```suggestion
    depends_on("llvm@11.0:14~flang", when="@0.40.0:")
    depends_on("llvm@11.0:11~flang", when="@0.37.0:0.39")
```

Reply:
Reported https://github.com/numba/llvmlite/issues/956

Reply:
Addressed in https://github.com/numba/llvmlite/pull/958, i.e. only LLVM 11 or 14 are supported for 0.40 though 12 and 13 may work. This will require another small modification to the dependencies to exclude LLVM 12 and 13.